OpenRA.Mods.Common/Traits/Carryall.cs

```diff
@@ -355,23 +355,23 @@ public override bool CanTargetFrozenActor(Actor self, FrozenActor target, Target
 			}
 		}
 
-		class CarryallDeliverUnitTargeter : AircraftMoveOrderTargeter
+		class CarryallDeliverUnitTargeter : IOrderTargeter
 		{
 			readonly AircraftInfo aircraftInfo;
 			readonly CarryallInfo info;
-			readonly WVec carryableOffset;
+
+			public string OrderID { get { return "DeliverUnit"; } }
+			public int OrderPriority { get { return 6; } }
+			public bool IsQueued { get; protected set; }
+			public bool TargetOverridesSelection(TargetModifiers modifiers) { return true; }
 
 			public CarryallDeliverUnitTargeter(AircraftInfo aircraftInfo, CarryallInfo info, WVec carryableOffset)
```

Comment:
`carryableOffset` appears to be unused now...?

Reply:
#16417 removed all uses of it. This just cleans up the unused variable as a part of the other changes.

Reply:
But why is it still included in the targeter then?

Reply:
Ah, I see. Fixed.